src/Utilities/model.jl

```diff
@@ -887,9 +887,9 @@ for (loop_name, loop_super_type) in [
 end
 
 const LessThanIndicatorSetOne{T} =
```

Comment:
A nit: did you search for "IndicatorSet" that didn't match the entire word? seems not? How many are there

Reply:
Good find! I renamed the four occurrences there. There should be none remaining, except for MOF: 

![image](https://user-images.githubusercontent.com/285818/127844735-7832dd5e-1fc1-411b-805d-4f1785268852.png)